### DISCUSSION TITLE
Valkey Enterprise Support

### DISCUSSION THREAD

**Discussion by JBrejnholt** [valkey-io/valkey#1381] [General]
Hi,

Are there any recommended Valkey Enterprise support out there?

**Comment by martinrvisser** [comment]
Hi,
At Percona we are starting to offer support from Mid December for Valkey.

**Reply by JBrejnholt** [reply]
Thanks.

I have received your email. I will reply from my work email.

**Comment by madolson** [comment]
The OSS Valkey community will never have an enterprise version, but as martin mentioned, there will be third parties who will provide enterprise suppose like Percona.

Just for completeness for people looking in the future, I believe Memurai also has an enterprise version: https://www.memurai.com/blog/memurai-valkey-for-windows-linux. I'm sure others will provide it in the future.
